packages/data-mate/src/function-configs/string/isMimeType.ts

```diff
@@ -0,0 +1,11 @@
+import { isMimeType } from '@terascope/utils';
+import { FieldValidateConfig, ProcessMode, FunctionDefinitionType } from '../interfaces';
+
+export const isMimeTypeConfig: FieldValidateConfig = {
+    name: 'isMimeType',
+    type: FunctionDefinitionType.FIELD_VALIDATION,
+    process_mode: ProcessMode.INDIVIDUAL_VALUES,
+    description: 'Checks to see if input is a MIME Type',
+    create() { return isMimeType; },
+    accepts: [],
```

Comment:
This should probably only accept strings since this could be optimized out if given anything else
```suggestion
    accepts: [FieldType.String],
```

Reply:
fixed this with latest commit